Cql/Cql.Packaging/LibraryPackager.cs

```diff
@@ -39,20 +39,14 @@ internal LibraryPackager(TypeConverter? typeConverter)
             TypeConverter = typeConverter ?? FhirTypeConverter.Create(ModelInfo.ModelInspector);
         }
 
-        public static IDictionary<string, elm.Library> LoadLibraries(DirectoryInfo elmDir)
-        {
-            var dict = new ConcurrentDictionary<string, elm.Library>();
-            var files = elmDir.GetFiles("*.json", SearchOption.AllDirectories);
-            Parallel.ForEach(files, file =>
-            {
-                var library = elm.Library.LoadFromJson(file);
-                if (library?.NameAndVersion != null)
-                {
-                    dict.TryAdd(library.NameAndVersion, library);
-                }
-            });
-            return dict;
-        }
+        public static IDictionary<string, elm.Library> LoadLibraries(DirectoryInfo elmDir) =>
```

Comment:
This is an unncessary change that makes the code harder to debug.  You will find throughout this SDK that I write code in a way that you can easily conclude, "But you can do that in just a couple of LINQ statements."  I used to write code like this proposed change, and then stopped doing it because I constantly ended up rewriting my code to look like the original function here because I wanted to put a break point somewhere and couldn't.

Reply:
The change was necessary because we need to have a deterministic outcome, by loading the input files in order. I changed it back to using the original Parallel.For, however the ordering of the files beforehand still uses linq though.